relay/engines/docker.go

```diff
@@ -59,54 +59,55 @@ func (de *DockerEngine) Init() error {
 
 // IsAvailable returns true/false if a Docker image is found
 func (de *DockerEngine) IsAvailable(name string, meta string) (bool, error) {
-	err := de.attemptAuth()
-	if err != nil {
-		return false, err
-	}
 	fullName := fmt.Sprintf("%s:%s", name, meta)
-	log.Debugf("Retrieving %s:%s from upstream Docker registry.", name, meta)
-	beforeID, _ := de.IDForName(name, meta)
-	closer, pullErr := de.client.ImagePull(context.Background(), fullName,
-		types.ImagePullOptions{
-			All:          false,
-			RegistryAuth: de.registryToken,
-		})
-	if closer != nil {
-		ioutil.ReadAll(closer)
-		closer.Close()
-	}
-	if pullErr != nil {
-		log.Errorf("Error ocurred when pulling image %s: %s.", name, pullErr)
-		image, _, inspectErr := de.client.ImageInspectWithRaw(context.Background(), fullName, false)
-		if inspectErr != nil {
-			log.Errorf("Unable to find Docker image %s locally or in remote registry.", name)
-			return false, pullErr
+	found := false
```

Comment:
I don't think the `found` variable is ever touched after this... if so, at least one level of nesting can be removed from the whole function.